router/internal/config/config.go

```diff
@@ -49,22 +63,43 @@ type Prometheus struct {
 	ListenAddr string `yaml:"listen_addr" default:"127.0.0.1:8088" validate:"hostname_port" envconfig:"PROMETHEUS_LISTEN_ADDR"`
 }
 
+type MetricsExporter struct {
+	Exporter otelconfig.Exporter `yaml:"exporter" validate:"oneof=otlphttp otlpgrpc"`
+	Endpoint string              `yaml:"endpoint" validate:"http_url"`
+	HTTPPath string              `yaml:"path"`
+	Headers  map[string]string   `yaml:"headers"`
+}
+
 type Metrics struct {
-	Common     MetricsCommon `yaml:"common"`
-	Prometheus Prometheus    `yaml:"prometheus"`
+	Common     MetricsCommon     `yaml:"common"`
+	Exporters  []MetricsExporter `yaml:"exporters"`
+	Prometheus Prometheus        `yaml:"prometheus"`
 }
 
 type MetricsCommon struct {
 	Enabled bool `yaml:"enabled" default:"true" envconfig:"METRICS_ENABLED"`
 }
 
-type OpenTelemetry struct {
-	ServiceName string            `yaml:"service_name" default:"cosmo-router" envconfig:"TELEMETRY_SERVICE_NAME" validate:"required"`
-	Endpoint    string            `yaml:"endpoint" validate:"required" default:"https://cosmo-otel.wundergraph.com" envconfig:"TELEMETRY_ENDPOINT" validate:"http_url"`
-	Headers     map[string]string `yaml:"headers" envconfig:"TELEMETRY_HEADERS"`
-
-	Tracing Tracing `yaml:"tracing"`
-	Metrics Metrics `yaml:"metrics"`
+// telemetry:
```

Comment:
Can we delete it?

Reply:
Sure, I `git add` ed it by mistake, sorry about that!